services/geostore.js

```diff
@@ -48,28 +64,35 @@ const fetchGeostore = ({ url, token }) =>
         bbox: geostore?.bbox,
       };
     });
+};
 
 export const getGeostore = ({ type, adm0, adm1, adm2, token }) => {
   if (!type || !adm0) return null;
 
-  const adminURL = `/geostore/admin/${adm0}${adm1 ? `/${adm1}` : ''}${
-    adm2 ? `/${adm2}` : ''
-  }`;
+  const sourceProvider = 'source[provider]=gadm';
+  const sourceVersion = 'source[version]=3.6';
+  const threshold = `simplify=${setThreshold(adm0, adm1, adm2)}`;
```

Comment:
@willian-viana, it looks to me that a threshold was only ever applied to a `country` geostore. 
This PR seems to add it to every case (`country`, `use`, and `geostore`). 

I'm thinking this could result in unexpected geometries being returned from the Data API Geostore service.

I recommend applying the `simplify` query parameter to `country` geostore URLs **only**. Unless you see something I'm missing (and that's extremely possible). 😄

Reply:
You are right. I can't see a user drawing an area larger than a country like Brazil. So we don't need this parameter. I'll remove it asap